docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
         - no_proxy
     image: hmcts/civil-sdt
     environment:
+        CIVIL_SDT_S2S_KEY: "AAAAAAAAAAAAAAAA" # use aat key from vault
```

Comment:
Does this need to be prefixed with "-" as other environment variables are?

Reply:
that is correct, changed it now